README.md

```diff
@@ -28,7 +28,12 @@ TODO: Write usage instructions here
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`.
+
+To release a new version, update the version number in `version.rb`, update the
+CHANGELOG.md and create a PR for this version change. Once this PR is merged,
+tag the merge commit with the version number, and push the tag. This will cause
```

Comment:
Good call. I've updated this to specify semver, and also to link out to the GDS rubygems styleguide for more details on this.